pkg/appgw/redirects.go

```diff
@@ -0,0 +1,44 @@
+package appgw
+
+import (
+	n "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+	"k8s.io/api/extensions/v1beta1"
+)
+
+func (builder *appGwConfigBuilder) getRedirectConfigurations(ingressList []*v1beta1.Ingress) *[]n.ApplicationGatewayRedirectConfiguration {
```

Comment:
This is currently dead code; with the next PR it will be used here:  https://github.com/Azure/application-gateway-kubernetes-ingress/pull/156/files#diff-2ef8906d6ce1592a472813bff3756698R14